utils/fileutils/fileutils_test.go

```diff
@@ -14,175 +14,6 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestFindConfigFile(t *testing.T) {
```

Comment:
These tests were basically duplicates of the `TestFindFile`, and are covered by existing tests in the `config` package.